### ISSUE TITLE
Text way too dark in DevTool dark theme

### ISSUE THREAD

**Issue by EWhite613** [dwickern/ember-chrome-devtools#1]
![screen shot 2017-07-05 at 10 56 02 am](https://user-images.githubusercontent.com/9057680/27870359-abbdff90-6170-11e7-9a5a-5e85ea8dfb0a.png)

Looks like it's just the keys that are using the light theme color scheme

**Comment by dwickern** [comment]
Yeah, unfortunately chrome doesn't tell me whether the dark theme is active or not. Custom formatters only have access to the `style` tag so you can't even add the CSS `class` that devtools already defines.

We could find some colors which work with both themes, or maybe we add some config setting in `ember-cli-build.js` which switches the addon to a dark theme

**Comment by dwickern** [comment]
Related binaryage/cljs-devtools#30

**Comment by EWhite613** [comment]
Is there an option to do CSS classes in custom formatter, rather than inline? Because the body of the dev console has the class `-theme-with-dark-background`. 

We could also alternatively check if that class is present on the body element (assuming there is some way to do a selector on the devtools)

**Comment by dwickern** [comment]
Nope, can't use any attributes except `style` (source: [devtools sources](https://github.com/ChromeDevTools/devtools-frontend/blob/dfa5242e212e67dde9a5c02165e25a578bfd85c2/front_end/object_ui/CustomPreviewComponent.js#L75)). And inline styles can't use css rules like `-theme-with-dark-background > div`.

We can't check if the class is present on the body element either, because our code runs on the webpage, not inside the devtools window.
